### DISCUSSION TITLE
How to use LDSR upscaler

### DISCUSSION THREAD

**Discussion by tzekeong** [comfyanonymous/ComfyUI#68] [Q&A] [answered]
Can we use LDSR upscaler in the project?

**Comment by comfyanonymous** [comment, marked as answer]
No that's not implemented. I plan on implementing at least one latent upscale model eventually.

**Reply by jonathanjmak** [reply]
Any updates on when it would be added?

**Comment by angrysky56** [comment]
Ah, I was wondering too, even though it is slow it gives nice results. Are there any comparable models?
I tried to move the files from my automatic1111 .cache lol.
P.S. I asked Vlad to get ComfyUI better integrated as a tab for his automatic fork with a library to share ideas built in. He and a few others were interested. Comfy needs dropdown tabs that you can insert anywhere and stretch and way better module menus. I love it so much though, thank you.

LDSR for automatic1111
https://github.com/Hafiidz/latent-diffusion

**Comment by PeggyFree** [comment]
Hello, as a happy user of portable ComfyUI, I humbly ask too for adding LDSR to ComfyUI.
Comfy deserves LDSR, I have no UI anymore to use the best upscaler ever.

**Comment by luxdeluks** [comment]
Just wanted to keep the interest alive as I found LDSR combined with ultimate upscale to create wonders in auto1111. Controlnet tile couldn't come close.

Thanks for all the work

**Comment by TomLucidor** [comment]
For some reference on LDSR (what the users want) vs ESRGAN (current plan) https://github.com/comfyanonymous/ComfyUI/issues/41 https://github.com/comfyanonymous/ComfyUI/issues/1242

**Comment by TomLucidor** [comment]
Has anyone mentioned 4x-AnimeSharp (from UltraMatrix) and Waifu2x (DCNN) yet? Also what other models can you guys think of? https://www.pixcores.com/2023/05/free-anime-upscaler-model
P.S. Pretty sure model-based nodes can be added for some functions but not sure about the issues of UpScalers https://github.com/pythongosssss/ComfyUI-WD14-Tagger

**Comment by alenknight** [comment]
came here to get further info on this.  would REALLY like to see LDSR in comfyui.

**Comment by ltdrdata** [comment]
https://github.com/flowtyone/ComfyUI-Flowty-LDSR

**Reply by alenknight** [reply]
just saw this yesterday too! yay it's working!

**Reply by stephantual** [reply]
yeah it's brilliant. I love it!

**Comment by ltdrdata** [comment]
In the previous version, there was an issue where parts were cropped and padded in black at resolutions that were not multiples of 64. If you were using the previous version, please update.
